server/controllers/card.js

```diff
@@ -44,34 +37,25 @@ exports.updateCard = asyncHandler(async (req, res, next) => {
     attachment,
     checklist,
     columnId,
-    index,
-    createdAt,
   } = req.body;
 
-  try {
-    const filter = { _id };
-    const update = {
-      _id,
-      name,
-      tag,
-      description,
-      deadline,
-      comment,
-      attachment,
-      checklist,
-      columnId,
-      index,
-      createdAt,
-    };
+  const filter = { _id };
```

Comment:
There really isn't a need to make a variable for this. It's not big enough, just pass it along to the `findOneAndUpdate`